privaterelay/views.py

```diff
@@ -97,6 +97,17 @@ def profile(request):
     return render(request, 'profile.html', context)
 
 
+def settings(request):
```

Comment:
This is missing the avatar context, which is not setting/rendering the FxA profile image correctly. This could be added to the context processor, or exposed like the FAQ view does: https://github.com/mozilla/fx-private-relay/blob/main/privaterelay/views.py#L61-L65

Broken: 
<img width="313" alt="image" src="https://user-images.githubusercontent.com/2692333/136417948-efa9ae43-38d9-4518-a0fc-c036856f7390.png">

Working: 
<img width="317" alt="image" src="https://user-images.githubusercontent.com/2692333/136417967-d355ca56-b941-4537-9b25-3d8be0da1b9f.png">

Reply:
Yup, fixed.